Sources/EduIDExpansion/Flows/Onboarding/OnboardingCoordinator.swift

```diff
@@ -0,0 +1,59 @@
+import UIKit
+
+final class OnboardingCoordinator: OnboardingCoordinatorType, OnBoardingNavigationDelegate, ScanMainNavigationDelegate {
```

Comment:
Same here: split up conformances to separate extensions.

Reply:
I did this